### DISCUSSION TITLE
Tnt And Creeper

### DISCUSSION THREAD

**Discussion by anukalp-k** [GriefPrevention/GriefPrevention#1433] [Q&A] [answered]
I wanted to allow Tnt explosion outside claims in all the worlds(overworld,nether and end) but disable creeper explosion in the worlds. I did mobgriefing off which lead to villagers not breeding... Pls help me And I have the sea level overide set to 256 in all dimesions or worlds

**Comment by RoboMWM** [comment, marked as answer]
don't touch sea level override and instead use the config options.

**Reply by anukalp-k** [reply]
Thx I got it yesterday while checking the config Thx
